### ISSUE TITLE
Cannot connect TP-LINK TL-WN722N

### ISSUE THREAD

**Issue by clearvision17** [offensive-security/kali-nethunter#1158]
### Device:

Nexus 7

### OS version (KitKat/Lollipop/Marshmallow/Nougat):

Android 6.0.1

### Built from repo (date and build command) or downloaded from website (links):

nethunter was installed from their official website

### Output of `cat /proc/version` in adb shell or NetHunter Terminal:

Linux version 3.4.0-gcc51ee3-dirty (root@kali) (gcc version 4.7 (GCC) ) #4 SMP PREEMPT Sat Dec 12 00:29:55 UTC 2015

## Issue:

cannot connect wireless adaptor to net hunter. any ideas guys i have been browsing for 8 hours straight and still found no solution

self diagnostics

**Comment by clearvision17** [comment]
if i could provide additional information please inform me

**Comment by ll3N1GmAll** [comment]
You are not alone. I have noticed other issues on this github with the exact same problem; and I too am experiencing this. I have tried with the exact same adapter that you mentioned as well as another which uses the rt2870/rt3070 chipset. LSUSB shows both adapters with their details when they are plugged in; but neither respond to and iw/ifconfig commands. They do not show as listed when iw/ifconfig are entered either.
